src/ImGui.NET.FNA/DrawVertXNA.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ImGuiNET;
+using Microsoft.Xna.Framework.Graphics;
+using System.Numerics;
+using System.Runtime.InteropServices;
+
+namespace ImGuiNET.FNA
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public unsafe struct DrawVertXNA
```

Comment:
I don't think this type is really necessary. It's structurally identical to the native vertex type, and the static stuff could just be helper functions in the renderer.